yt/utilities/command_line.py

```diff
@@ -1477,30 +1477,30 @@ def __call__(self, args):
         ds = args.ds
         ds.print_stats()
         vals = {}
-        if args.field in ds.derived_field_list:
-            if args.max:
-                vals["min"] = ds.find_max(args.field)
-                print(
-                    f"Maximum {args.field}: {vals['min'][0]:0.5e} at {vals['min'][1]}"
-                )
-            if args.min:
-                vals["max"] = ds.find_min(args.field)
-                print(
-                    f"Minimum {args.field}: {vals['max'][0]:0.5e} at {vals['max'][1]}"
-                )
+        field = ds._get_field_info(args.field)
+        if args.max:
```

Comment:
It looks like the logic is broken here, isn't it?

Reply:
Sorry, could you elaborate? Worked for me after I made this change.

Reply:
I'm confused as well. For instance the line after this one
```python
vals["min"] = ds.find_max(field)
```
_looks_ broken. Maybe it's not, but in any case I don't understand it as it is

Reply:
Ah dang--I didn't look at it carefully enough. Yeah, I'll fix it.

Reply:
thanks !